SUPPORT_POLICY.md

```diff
@@ -0,0 +1,27 @@
+Distroless is designed to follow debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
+
+The current estimation of EOL of images
```

Comment:
Is it worth spelling out that this is for specific tags (or patterns of tags). This table is hard to make sense of if I just use `:latest`

Reply:
how's this look?